websites/G/Genshin Impact Map/presence.ts

```diff
@@ -0,0 +1,166 @@
+const presence = new Presence({
+		clientId: "973729201104511007"
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+interface Maps {
```

Comment:
```suggestion
interface Map {
```

Reply:
cannot ,don't change it any more u make it inactive

Reply:
you have to change this in your IDE along with all the references to it, you're not meant to merge this suggestion simply

Reply:
This has not been addressed